src/main/java/hudson/plugins/git/extensions/impl/impl.plantuml

```diff
@@ -0,0 +1,956 @@
+@startuml
+
+title __IMPL's Class Diagram__\n
+
+  namespace  {
+    namespace udson.plugins.git {
```

Comment:
If you're storing a UML representation like this, shouldn't the namespace match the package name?
```suggestion
    namespace hudson.plugins.git {
```